api/answer/ConceptMap.java

```diff
@@ -35,4 +40,26 @@
 
     @CheckReturnValue
     Concept get(String variable);
+
+    Explainables explainables();
+
+    interface Explainables {
+
+        Explainable concept(String variable);
+
+        Explainable ownership(String owner, String attribute);
+
+        Map<String, Explainable> explainableConcepts();
+
+        Map<Pair<String, String>, Explainable> explainableOwnerships();
+
+        interface Explainable {
+
+            String conjunction();
```

Comment:
What is an Explainable's `conjunction`?

Reply:
It's the corresponding part of the user's original query (ie the sub-conjunction) that is being explained, mostly for the user to see if they are interactively querying this api

Reply:
Do you think `subquery`, or `queryPart`, or `queryFragment` might be a clearer name? `conjunction` and `disjunction` are very logical terms.

Reply:
"Explainable Conjunction" is alright i think, the user will see its a subquery pretty quickly